devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -93,6 +97,13 @@ def updated() {
 	response(cmds)
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
```

Comment:
Are you able to get any event out of this device with a `batteryGet()`?
My intuition is that it is sleepy and won't respond.